Makefile

```diff
@@ -5,16 +5,19 @@ TIMESTAMP=$(shell date -u +%Y%m%d.%H%M%S)
 BUILD_FLAGS=-mod=readonly -ldflags="-X main.version=$(VERSION)-$(TIMESTAMP)"
 .DEFAULT_GOAL := build
 
+THEMIS_INSTALL_DIR ?= /usr/local/bin/
+
 all: build test
 
 build:
-	cd pkg/themis-contract/ && statik -src=../../assets/
+	cd pkg/themis-contract/ && statik -f -src=../../assets/
 	go build $(BUILD_FLAGS) -o $(OUTPUT) cmd/themis-contract/*
 
 test:
 	go test $(PACKAGES)
 
 install: build
-	cp $(OUTPUT) /usr/local/bin/
+	go get github.com/rakyll/statik
```

Comment:
Shouldn't this line be moved to before the `statik` call on line 13? (since `install` depends on `build`)

Also, in general, it's probably not the greatest idea to install dependencies on every build. Ideally we'd have the Makefile autodetect whether or not `statik` is installed and then install it if necessary, but a suitable compromise could be to have a build target that installs dependencies that needs to be run by the user once off, like:

```Makefile
install-deps:
    go get github.com/rakyll/statik

.PHONY: all build test install install-deps
```

And then in the installation from source docs:

```bash
# Run once-off
make install-deps
# Run to build and install Themis Contract
make install
```

Reply:
Ah yeah. Not sure what I was thinking with that recipe. Fixed!